src/components/atomic/Autocomplete/Autocomplete.jsx

```diff
@@ -344,9 +344,7 @@ class Autocomplete extends React.Component {
 			/*if (!this.isOpen()) {
 				// menu is closed so there is no selection to accept -> do nothing
 				return;
-			} else */ if (
-				this.state.highlightedIndex == null
-			) {
+			} else */ if (this.state.highlightedIndex == null) {
```

Comment:
Can this commented block be totally removed? You can add appropriate comment to the commit so it can be referenced if a reversion is needed in the future.